### PR TITLE
Modify cite element to use a variable for the pseudo content

### LINKED ISSUE #8845: Add setting variable for cite ::before content
Can the content for the ::before for the cite element in typography/_base.scss be specified with a variable, so that we can set an override?

Yes, I can just change it easily in my CSS, but it'd be nice to just be able to set a variable and have the default style be what I want from the start.

### PR DESCRIPTION
**Pull request by nessthehero**
Adds an override variable to specify the cite element's pseudo ::before content.

This pull request resolves #8845

### CODE REVIEW

File: scss/typography/_base.scss [hunk 1]
```diff
@@ -203,6 +203,10 @@ $cite-font-size: rem-calc(13) !default;
 /// @type Color
 $cite-color: $dark-gray !default;
 
+/// Pseudo content for `<cite>` elements.
+/// @type String
+$cite-pseudo-content: '\2014 \0020';
```
Comment (denisahac) [inline scss/typography/_base.scss]: Append the **!default** annotation, and declare the same variable in `settings/_settings.scss` to allow anyone to override the default value.

Comment (nessthehero) [inline scss/typography/_base.scss]: Thanks, I made this change.

Comment (nessthehero) [inline scss/typography/_base.scss]: Oh wait didn't notice the settings.scss mention. On route.

Comment (nessthehero) [inline scss/typography/_base.scss]: Hmm, that file is empty. Am I missing something?

**Comment by nessthehero** [comment]
Oh dear, the check failed.

**Comment by andycochran** [comment]
Should this variable also be added to `_settings.scss`? If so, the PR should target the 6.3 branch.

**Comment by nessthehero** [comment]
Ok, should I close this PR and create a new one targeted to the 6.3 branch?

**Comment by rafibomb** [comment]
Sorry for the delay - yes `v6.3` 

So **6.2.4** (minor release) is for general bug fixes
**6.3** will be new features (variables, JS options, classes etc)

**Comment by nessthehero** [comment]
I merged the latest 6.3 into my branch, but I want to confirm if I need to close this pull request and create a new one, targeting the marge into the 6.3 branch itself. Or is this fine?

**Comment by andycochran** [comment]
Yup, you should probably close this one and make your changes on a fresh checkout of the 6.3 branch so that the PR only includes your work.

**Comment by nessthehero** [comment]
Will do! Thanks!

**Pull request closed without merging**